app/javascript/components/pages/Home.jsx

```diff
@@ -5,59 +5,93 @@ import Button from 'components/Button';
 import PageTitle from 'components/PageTitle';
 import PodcastTile from 'components/PodcastTile';
 import InfoCard from '../home/InfoCard';
-import { infoCardData } from '../home/homeData';
 import SponsorsTile from 'components/SponsorsTile';
 
 import 'stylesheets/page';
 import 'stylesheets/home';
 
 import PodcastImage from 'images/podcast-cover.png';
-import SponsorImage1 from 'images/sponsor1.png';
-import SponsorImage2 from 'images/sponsor2.png';
-import SponsorImage3 from 'images/sponsor3.png';
+import Stripe from 'images/sponsors/stripe.svg';
+import Shopify from 'images/sponsors/shopify.svg';
+import Contentful from 'images/sponsors/contentful.svg';
 
-let podcast = {
+import JoinOurWelcomingCommunitySection from '../home/JoinOurWelcomingCommunitySection';
+import ExceedYourProfessionalGoalsSection from '../home/ExceedYourProfessionalGoalsSection';
+import GiveSupportSection from '../home/GiveSupportSection';
+import JoinCommunityIcon from 'components/icons/JoinCommunity';
+import ExceedGoalsIcon from 'components/icons/ExceedGoals';
+import GiveSupportIcon from 'components/icons/GiveSupport';
+
+const podcast = {
     tile_image: PodcastImage,
     tile_image_alt: 'Ruby on Rails Podcast',
     title: 'Ruby on Rails Podcast',
     url: 'https://www.therubyonrailspodcast.com/373',
 };
 
-let sponsors = [
+const sponsors = [
     {
         id: 1,
-        tile_image: SponsorImage1,
-        tile_image_alt: 'Sponsor 1',
+        tile_image: Stripe,
+        tile_image_alt: 'Stripe',
     },
     {
         id: 2,
-        tile_image: SponsorImage2,
-        tile_image_alt: 'Sponsor 2',
+        tile_image: Shopify,
+        tile_image_alt: 'Shopify',
     },
     {
         id: 3,
-        tile_image: SponsorImage3,
-        tile_image_alt: 'Sponsor 3',
+        tile_image: Contentful,
+        tile_image_alt: 'Contentful',
+    },
+];
+
+const infoCardData = [
+    {
+        title: 'Join our welcoming community',
+        section: <JoinOurWelcomingCommunitySection />,
+        icon: JoinCommunityIcon,
+    },
+    {
+        title: 'Exceed your professional goals',
+        section: <ExceedYourProfessionalGoalsSection />,
+        icon: ExceedGoalsIcon,
+    },
+    {
+        title: 'Give support, and get it in return',
+        section: <GiveSupportSection />,
+        icon: GiveSupportIcon,
     },
 ];
 
 const Home = () => (
     <SharedLayout>
-        <section className="hero">
-            <PageTitle text="WNB.rb">
-                <p>A virtual community for women and non-binary Rubyists.</p>
-                <a href="https://tinyurl.com/join-wnb-rb" target="_blank" rel="noreferrer noopener">
-                    <Button type="secondary">Join WNB.rb</Button>
-                </a>
-            </PageTitle>
-            <div className="hero-right">
-                <SplashBackground className="splash-background" />
+        <section className="hero-container">
```

Comment:
I cleaned up the styles for this section, allowing the WNB.rb title and the splash image to wrap when the screen gets small!